job_executor/app.py

```diff
@@ -39,6 +37,44 @@ def logger_thread(logging_queue: Queue):
         logger.handle(record)
 
 
+def fix_interrupted_jobs():
+    logger.info('Querying for interrupted jobs')
+    in_progress_jobs = job_service.get_jobs(ignore_completed=True)
+    queued_statuses = ['queued', 'built']
```

Comment:
Why check the queued jobs? What happens later on line 59 is setting the same 'queued' status.

Reply:
these queued statuses are used to filter out jobs with the queued statuses on line 46 (as these have not been interrupted).
Line 59 sets the status of the job to queued, but since we only process each item in the list once, the function won't stumble on an updated job 👍🏻